StatusCodes.md

```diff
@@ -0,0 +1,47 @@
+# HTTP Status Codes
+## Complete list of Status Codes
+A full list of HTTP Status Codes with explanation and related RFC specs can be found on [Wikipedia](https://en.wikipedia.org/wiki/List_of_HTTP_status_codes).
+
+## Common used codes within Digipolis
+This is a list of common used status codes that should be handled correctly by clients consuming services approved by the Digipolis REST API Guidelines Working Group.
+
+Each API method should preferably document the status codes that can be returned in the Swagger documentation.
+
+### 2xx (Successful)
+The request was successfully received, understood, and accepted.
+
+| Code | Status     | When to use
+|------|------------|--------------
+| 200  | OK         | Standard response for successful HTTP requests.
+| 201  | Created    | The request resulted in the creation of a new resource.
+| 202  | Accepted   | The request has been accepted for processing (long running job).
+| 204  | No Content | The server successfully processed the request and is not returning any content.
+
+### 3xx (Redirection)
+Further action needs to be taken in order to complete the request.
+
+| Code | Status            | When to use
+|------|-------------------|--------------
+| 301  | Moved Permanently | This and all future requests should be directed to the given URI.
+| 307  | Temporary Redirect | In this case, the request should be repeated with another URI; however, future requests should still use the original URI.
```

Comment:
Do we want to indicate that a 302 shouldn't be used because of the ambiguous behaviour (that resulted in 303 and 307 being added)?

Reply:
-Sorry to barge in, but this conversation popped up in my mail- 
What about 308 vs 301?

Reply:
¯\_(ツ)_/¯

I just tried to be smart with 307. But since we're talking about APIs, the HTTP method should not change, so 308 might be the better option as well. And mention that these should be used over 301/302.

Reply:
That seems fine :)
Maybe a line for 301/302 with the other code that should be used instead in the When to use field?